homeassistant/components/purpleair/sensor.py

```diff
@@ -166,15 +166,14 @@ class PurpleAirSensorEntityDescription(
         name="Uptime",
         entity_category=EntityCategory.DIAGNOSTIC,
         entity_registry_enabled_default=False,
-        icon="mdi:timer",
+        device_class=SensorDeviceClass.DURATION,
         native_unit_of_measurement=UnitOfTime.MINUTES,
         state_class=SensorStateClass.TOTAL_INCREASING,
         value_fn=lambda sensor: sensor.uptime,
     ),
     PurpleAirSensorEntityDescription(
         key="voc",
         name="VOC",
-        device_class=SensorDeviceClass.VOLATILE_ORGANIC_COMPOUNDS,
         native_unit_of_measurement=CONCENTRATION_IAQ,
```

Comment:
Why don't we use the air quality device class if the measurement is expressed in the air quality index?

Reply:
Not sure I understand your question. This isn't AQI; IAQ is a different unit.

Reply:
According to the docs you linked they calculate the IAQ index 0-500 for the VOC measurement of the BME680. Am I misinterpreting it?

Reply:
This comment? (emphasis mine)

> In general,  BME680 is detect the TVOC component and the register value is changing based on the concentration of those component.    BSEC lib will be used to integrated the output of gas resgister value and convert it to **IAQ index which defined as 0 - 500**.   BSEC lib will track the baseline of the VOC value and set the cleanest environment as IAQ 0.

It isn't clear what the relationship between IAQ and AQI is. [This article](https://community.bosch-sensortec.com/t5/MEMS-sensors-forum/BME680-strange-IAQ-and-CO2-values/m-p/9667/highlight/true#M1505) describes a range different from AQI. What constitutes picking a device class in a scenario like this?

Reply:
Table 4 here seems to describe an AQI:
https://www.bosch-sensortec.com/media/boschsensortec/downloads/datasheets/bst-bme680-ds001.pdf

It doesn't match exactly to any of the different [country specific AQI:s](https://en.wikipedia.org/wiki/Air_quality_index), but it's similar.

We haven't described exactly which index should be followed for AQI device class as there seems to be a plethora. But if a AQI measurement has a range between 0-500 and similar category limits I'd approve it for now. Of course it would be good if we described this better in the future.